docs/getting_started/basic/zero-shot-prediction/zero-shot-classification.ipynb

```diff
@@ -0,0 +1,479 @@
+{
```

Comment:
<p>"You can also run Hugging Face Transformer models from Hugging Face..."</p><p> </p><p>? Note: these models must be supported in the Hugging Face Transformers library which are listed here[link]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thesteve0/v51-gs-draft/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>